src/scripts/main.js

```diff
@@ -1,3 +1,27 @@
 'use strict';
 
-// write your code here
+const totalPopulation = document.querySelector('.total-population');
+const averagePopulation = document.querySelector('.average-population');
+const population = document.querySelectorAll('.population');
+
+function totalSum(numbers) {
```

Comment:
Function names are typically verbs or verb phrases.
```suggestion
function countTotalSum(numbers) {
```